study.md

```diff
@@ -43,21 +47,26 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+View seems to mean something like what I thought "page" was. It's the way a rendered page looks in a specific instance. Front-end "routers" manage this view-state by hiding and showing certain elements of the html at times to change the look of the page without refreshing and re-rendering the full code.
+http://singlepageappbook.com/goal.html
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+Applications are more fluid and responsive--there is no need to reload pages every time a change is made. It also separates back-end and front-end work so that it's easier to completely change one without needing to change the other.
+https://msdn.microsoft.com/en-us/magazine/dn463786.aspx,
```

Comment:
Correct, but the backend and frontend can also be separated when not using an SPA also.